recipe/build_clangdev.sh

```diff
@@ -1,17 +1,16 @@
 if [[ "$(uname)" == "Linux" && "$cxx_compiler" == "gxx" ]]; then
-    sed -i.bak -e 's@addSystemInclude(DriverArgs, CC1Args, SysRoot + "/usr/local/include");@addSystemInclude(DriverArgs, CC1Args, "'"${PREFIX}/${HOST}/sysroot/usr/include"'");@g' \
```

Comment:
Note to myself for later: `SysRoot + "/usr/local/include"` needs to be replaced with `SysRoot + "/usr/include"`.